pkg/ec2helper/ec2helper.go

```diff
@@ -1478,6 +1514,12 @@ func (h *EC2Helper) LaunchFleet(templateId *string) (*ec2.CreateFleetOutput, err
 			fmt.Println(err.Error())
 		}
 		return nil, err
+	} else {
+		if len(result.Errors) != 0 {
+			err = errors.New(*result.Errors[0].ErrorMessage)
+			cli.ShowError(err, "Creating spot instance failed")
+			return nil, err
+		}
```

Comment:
Can you also add this code to the open [PR to add spot support](https://github.com/awslabs/aws-simple-ec2-cli/pull/85) on the main repo?

Reply:
How do I do that?

Reply:
I left a "suggestion"-style comment on the other PR that includes it. You should be able to accept that suggestion and then test to make sure it still works (e.g., pass an empty region, since that's the error we were seeing yesterday).

https://github.com/awslabs/aws-simple-ec2-cli/pull/85#discussion_r932828364